vochain/genesis/types.go

```diff
@@ -11,31 +11,31 @@ import (
 	"go.vocdoni.io/dvote/types"
 )
 
-// VochainGenesis is a struct containing the genesis details.
-type VochainGenesis struct {
+// Vochain is a struct containing the genesis details.
+type Vochain struct {
 	AutoUpdateGenesis bool
 	SeedNodes         []string
 	CircuitsConfigTag string
-	Genesis           *GenesisDoc
+	Genesis           *Doc
 }
 
 // The genesis app state types are copied from
 // github.com/cometbft/cometbft/types, for the sake of making this package
 // lightweight and not have it import heavy indirect dependencies like grpc or
 // crypto/*.
 
-// GenesisDoc defines the initial conditions for a Vocdoni blockchain.
+// Doc defines the initial conditions for a Vocdoni blockchain.
 // It is mostly a wrapper around the Tendermint GenesisDoc.
-type GenesisDoc struct {
+type Doc struct {
 	GenesisTime     time.Time        `json:"genesis_time"`
 	ChainID         string           `json:"chain_id"`
 	ConsensusParams *ConsensusParams `json:"consensus_params,omitempty"`
 	AppHash         types.HexBytes   `json:"app_hash"`
-	AppState        GenesisAppState  `json:"app_state,omitempty"`
+	AppState        AppState         `json:"app_state,omitempty"`
 }
 
 // TendermintDoc returns the Tendermint GenesisDoc from the Vocdoni GenesisDoc.
```

Comment:
actually i just noticed you should update the comment as well, as in:
```
// TendermintDoc returns the Tendermint GenesisDoc from the Vocdoni genesis.Doc
```

there are more ocurrences of this in the file

Reply:
i meant this

https://github.com/vocdoni/vocdoni-node/pull/1104/commits/c100eec27d99cc98b80dd579756b4be73a972a1a

(squash it before merging)